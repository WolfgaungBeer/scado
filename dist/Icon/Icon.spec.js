'use strict';

var _react = require('react');

var _react2 = _interopRequireDefault(_react);

var _reactTestRenderer = require('react-test-renderer');

var _reactTestRenderer2 = _interopRequireDefault(_reactTestRenderer);

var _Wrapper = require('../../__mocks__/Wrapper');

var _Wrapper2 = _interopRequireDefault(_Wrapper);

var _ = require('./');

var _2 = _interopRequireDefault(_);

function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { default: obj }; }

it('renders the Icon Component correctly', function () {
    var tree1 = _reactTestRenderer2.default.create(_react2.default.createElement(
        _Wrapper2.default,
        null,
        _react2.default.createElement(_2.default, { icon: 'check_circle' })
    )).toJSON();
    expect(tree1).toMatchSnapshot();

    var tree2 = _reactTestRenderer2.default.create(_react2.default.createElement(
        _Wrapper2.default,
        null,
        _react2.default.createElement(_2.default, { icon: 'check_circle', color: 'success', scale: 'l' })
    )).toJSON();
    expect(tree2).toMatchSnapshot();

    var tree3 = _reactTestRenderer2.default.create(_react2.default.createElement(
        _Wrapper2.default,
        null,
        _react2.default.createElement(_2.default, { icon: 'check_circle', color: 'success', scale: 'l', loading: true })
    )).toJSON();
    expect(tree3).toMatchSnapshot();
});