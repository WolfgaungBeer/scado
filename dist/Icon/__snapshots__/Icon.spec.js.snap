// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the Icon Component correctly 1`] = `
<i
  className="material-icons  sc-eNQAEJ dnHlAg"
  color={undefined}
  onClick={undefined}
  scale={undefined}
>
  check_circle
</i>
`;

exports[`renders the Icon Component correctly 2`] = `
<i
  className="material-icons  sc-eNQAEJ dbfNZB"
  color="success"
  onClick={undefined}
  scale="l"
>
  check_circle
</i>
`;

exports[`renders the Icon Component correctly 3`] = `
<i
  className="material-icons  sc-eNQAEJ hlKRCz"
  color="success"
  onClick={undefined}
  scale="l"
>
  autorenew
</i>
`;
